packages/frontend/src/components/profile/two_factor/TwoFactorAuth.js

```diff
@@ -64,41 +65,71 @@ const Container = styled(Card)`
     }
 `;
 
-const TwoFactorAuth = ({ twoFactor, history }) => {
+const TwoFactorAuth = ({ twoFactor, history, isBrickedAccount }) => {
     const [confirmDisable, setConfirmDisable] = useState(false);
+    const [showBrickedAccountModal, setShowBrickedAccountModal] = useState(false);
     const account = useSelector(selectAccountSlice);
     const existingContract = !ALLOW_2FA_ENABLE_HASHES.includes(account?.code_hash);
     const nearTokenFiatValueUSD = useSelector(selectNearTokenFiatValueUSD);
     const dispatch = useDispatch();
     const confirmDisabling = useSelector((state) => selectActionsPending(state, { types: ['DISABLE_MULTISIG'] }));
 
     const handleConfirmDisable = async () => {
-        await dispatch(disableMultisig());
+        if (isBrickedAccount) {
+            setShowBrickedAccountModal(true);
+        } else {
+            await dispatch(disableMultisig());
+        }
         await dispatch(fetchRecoveryMethods({ accountId: account.accountId }));
         setConfirmDisable(false);
     };
 
+    const onAccountLockClose = () => {
+        setShowBrickedAccountModal(false);
+    };
+
+    const onAccountLockComplete = () => {
+        setShowBrickedAccountModal(false);
+        this.forceUpdate();
```

Comment:
code breaks at this point. I don't see any method of `forceUpdate`

Reply:
This is an internal method for classComponents which we do not need anymore. There is a slight issue of the screen not refreshing once the 2fa has been disabled for a bricked account. I'll remove this, but that issue still stands. We can tackle it in next release.